src/components/Footer/Footer.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { NavLink } from 'react-router-dom';
+import './footer.css';
+import github from '../../images/gh.svg'
+import facebook from '../../images/fb.svg'
+
+
+function Footer(){
+    return(
+        <footer className="footer">
+      <p className="footer__copyright">
+        © {new Date().getFullYear()} Daniel Evgrafov
+      </p>
+      <nav className="footer__nav">
+      <div className="footer__links">
+      <NavLink
+            to= "/" 
+            className="footer__link"
```

Comment:
*  You need to format the markup to show indentation and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`